stack.yaml

```diff
@@ -41,6 +41,8 @@ extra-deps:
   commit: c03d17d656ac467350c983d5f844c199e5daceea    # master (Feb 21, 2019)
 - git: https://github.com/wireapp/hscim
   commit: 42f6018812bf0f04741231b67b1f5e790ce0d489    # master (Feb 25, 2019)
+- git: https://gitlab.com/fisx/tinylog
+  commit: ee39f436247e0c7ecd64ff7c1b28f3378179abfe    # https://gitlab.com/twittner/tinylog/merge_requests/4
```

Comment:
Guess this can be changed now?

Reply:
it's an independent change and (unless we make the new config fields optional, i'd have to check what the other services do there), it complicates the PR because it requires changes in our ops repos.

why not do it separately?

Reply:
I guess I misunderstood this comment

>  get https://gitlab.com/twittner/tinylog/merge_requests/4 merged, or move its contents to our extended package. bump tinylog dep to https://gitlab.com/twittner/tinylog/merge_requests/4

And reg.:

> requires changes in our ops repos.

What does that mean?

Reply:
tinylog merge request 4 *may* be going in soon, but it may take a while, or never happen.  i didn't want to wait for that; and leaving the comment in stack.yaml means that it'll be easy to follow up on changes upstream, should they ever happen.  then we can either pull in the changes that don't make it upstream, or just bump the dependency again to something more official.

Reply:
if you meant "change cargohold to support config of tinylog like the others": *that* would require to change other repos in order for ci / deployment to work.  i think.  unless it's all optional config fields.